<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextBook</class>
 <widget class="QWidget" name="TextBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Weight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Italic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FortGroundColor">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>前景色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackGroundColor">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>背景色</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_FontSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>字号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Font">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>字体：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
